content/theme.ts

```diff
@@ -30,6 +30,86 @@ const Progress: ComponentMultiStyleConfig = {
     }),
   },
 };
+const Button: ComponentStyleConfig = {
+  baseStyle: {
+    fontWeight: 700,
+    borderRadius: 'base',
+    color: 'black',
+    fontFamily: 'heading',
+    lineHeight: '20px',
+  },
+  variants: {
+    connect: {
+      bgGradient: chakraTheme.gradients['green-cyan']['500'],
+      _hover: {
+        bgGradient: chakraTheme.gradients['green-cyan']['600'],
+      },
+      _active: {
+        bgGradient: chakraTheme.gradients['green-cyan']['700'],
+      },
+    },
+  },
+};
+
+const Text: ComponentStyleConfig = {
+  variants: {
+    nav: {
+      fontWeight: 600,
+      color: 'whiteAlpha.800',
+      fontSize: '14px',
+      fontFamily: 'Inter',
```

Comment:
If we use `body` or `heading` the inter font renders without any problems 🤔

Reply:
yeah the way chakra is doing is that expects a prop of `fontFamily` that is either a key of the fonts object in the theme object. Bit confusing.

Reply:
Yeah it's a bit Idiosyncratic but good to know